tests/utils/dockercli/volumelifecycle.go

```diff
@@ -68,6 +68,24 @@ func AttachVolumeWithRestart(ip, volName, containerName string) (string, error)
 		dockercli.TestContainer)
 }
 
+// AttachWriteVolumeWithRestart Attach a volume with restart flag. Also create a file with specified name on volume
```

Comment:
I would suggest to elaborate more .. the intention behind adding this new util .. what does it do? or when to use?